modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -98,7 +98,8 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         DNN_TARGET_OPENCL,
         DNN_TARGET_OPENCL_FP16,
         DNN_TARGET_MYRIAD,
-        DNN_TARGET_FPGA  //!< FPGA device with CPU fallbacks using Inference Engine's Heterogeneous plugin.
+        DNN_TARGET_FPGA,  //!< FPGA device with CPU fallbacks using Inference Engine's Heterogeneous plugin.
+        DNN_TARGET_ARM,
```

Comment:
I believe `DNN_TARGET_CPU` should be reused instead.

Can we collect some extra information through InferenceEngine Query API to detect ARM CPU plugin?

Reply:
For ARM plugin, we need to skip some tests and layers. If we we are going to use  `DNN_TARGET_CPU`, we need to add extra information (conditions) to do this.

Reply:
We have such conditions for NCS / NCS2: `getInferenceEngineVPUType()`